src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.figure.Figure;
+import core.basesyntax.tools.FigureSupplier;
+
+public class Main {
+    private static FigureSupplier figureCreater = new FigureSupplier();

```

Comment:
```suggestion
    private static FigureCreator figureCreator = new FigureSupplier();
```

it is good practice to use interface (a more abstract type) as variable type